packages/dds/tree/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -128,6 +127,22 @@ export function isConflicted(mark: CanConflict): mark is Conflicted {
 	return mark.conflictsWith !== undefined;
 }
 
+export function cloneMark<TMark extends Mark<TNodeChange>, TNodeChange>(mark: TMark): TMark {
+	if (isSkipMark(mark)) {
+		return mark;
+	}
+	const objMark = mark as Exclude<TMark, Skip>;
```

Comment:
I kinda hate this cast is needed, Uf toy use 

```
export function isNotSkipMark<TMark extends Mark<unknown> | Skip>(
	mark: TMark | Skip,
): mark is Exclude<TMark, Skip> {
	return typeof mark !== "number";
}
```

then the need for it goes away. Not sure thats any better, but it is an alternative.